tests/test_mt_bench_model_adapter.py

```diff
@@ -0,0 +1,61 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Standard
+from unittest import mock
+
+# First Party
+from instructlab.eval.mt_bench_model_adapter import (
+    GraniteAdapter,
+    MistralAdapter,
+    get_conversation_template,
+    get_model_adapter,
+)
+
+MISTRAL_DEFAULT_MODEL_NAME = "mistral"
+EXAMPLE_MISTRAL_MODEL_PATHS = [
+    "mistral",
+    "mistralai/Mixtral-8x7B-Instruct-v0.1",
+    "/cache/instructlab/models/mistral-7b-instruct-v0.2.Q4_K_M.gguf",
+    "prometheus-eval/prometheus-8x7b-v2.0",
+    "/cache/instructlab/models/prometheus-eval/prometheus-8x7b-v2.0",
+]
+
+GRANITE_DEFAULT_MODEL_NAME = "granite"
+EXAMPLE_GRANITE_MODEL_PATHS = [
+    "granite",
+    "instructlab/granite-7b-lab",
+    "/cache/instructlab/models/instructlab/granite-7b-lab.gguf",
+    "instructlab/granite-8b-lab",
+]
+
+TEST_TUPLES = [
+    (
+        MISTRAL_DEFAULT_MODEL_NAME,
+        EXAMPLE_MISTRAL_MODEL_PATHS,
+        MistralAdapter,
+        MISTRAL_DEFAULT_MODEL_NAME,
+    ),
+    (
+        GRANITE_DEFAULT_MODEL_NAME,
+        EXAMPLE_GRANITE_MODEL_PATHS,
+        GraniteAdapter,
+        "ibm-generic",
+    ),
+]
+
+
+def test_get_model_adapter():
```

Comment:
Are we going to want to test that a `ValueError` gets thrown if `get_model_adapter()` doesn't have a valid model adapter for the given model path?

Reply:
Added, thanks!